### DISCUSSION TITLE
Detailed error message on deploy

### DISCUSSION THREAD

**Discussion by vkalach** [basecamp/kamal#586] [Q&A] [answered]
Hey, guys! I'm trying to deploy an app but during the deployment process app couldn't start that's why healthcheck fails. But, I can't understand what is the error - all I can see is cut (check the screenshot). How can I see the full trace of the error during the deployment?

![image](https://github.com/basecamp/kamal/assets/27734735/b00dffa8-e4d3-444b-81f9-5fa95461605b)

**Comment by vkalach** [comment]
I mean, I did it manually by connecting to the server and reproducing the run command for docker image, but maybe we can somehow tail logs during the deployment? 

-v didn't help

**Reply by nexneo** [reply]
How did you managed to get full log? I think I am facing same issue

**Reply by nexneo** [reply]
Got it, ran command from kamal output without `--detach`

**Comment by tsvallender** [comment, marked as answer]
Just hit this myself and discovered there’s an undocumented piece of configuration to increase log lines:
```yaml
healthcheck:
  log_lines: 1000
```

**Reply by nickhammond** [reply]
It looks like it's still in the docs but it's mentioned as the very last thing on this [healthchecks doc page](https://kamal-deploy.org/docs/configuration/healthchecks/). I put in a PR to include an example of using `log_lines` in that very first example https://github.com/basecamp/kamal-site/pull/70.
